docs/CONTRIBUTING.md

```diff
@@ -2,18 +2,100 @@
+* [Setting up Miunie](https://github.com/discord-bot-tutorial/Miunie#getting-started) - _Explains how to get a development version ready._
 * [First time contributing](https://egghead.io/courses/how-to-contribute-to-an-open-source-project-on-github) - _Free course walking you through your first contribution._
 * [Discord Server](https://discord.gg/cGhEZuk) - _Always someone ready to help on our Discord Server._
 * [DSharpPlus Docs](https://dsharpplus.github.io/) - _Documentation for the library we're using._
 
 ## How to submit changes
 
-If you're new to GitHub, we highly suggest you check out the **First time contributing** link in the [Important resources](#important-resources) section. It explains everything from committing your changes to creating a pull request.
+![Forking Miunie](https://i.imgur.com/mz93Muk.png)
+
+> Forking is a way of collecting all of the code and other good stuff we have in this repository and making a new repository off of it that is just yours to play with.
+>
+> You are fully in charge of your fork and don't need to worry about making any change you have in mind.
+
+* To fork Miunie, you will need a GitHub account.
+
+* Once you are logged in, just click the "Fork" button near the top of the page:
+
+![Fork button](https://i.imgur.com/oNViGmX.png)
+
+Once you have forked the repository, you can view it on your profile page under "Repositories".
+
+If you're not sure whether you're looking at a fork or the original, look near the top of the page where the name of the repository is.
+
+You can be sure you're looking at your fork if there is your name in front of ` / Miunie` and if it has a little "Forked from..." message under it.
+
+![Fork message](https://i.imgur.com/gjJHKfQ.png)
+
+After your fork is ready, you are free to [clone it to your local machine](https://help.github.com/en/articles/cloning-a-repository).
+
+![Keeping forks up to date](https://i.imgur.com/9XsurGf.png)
+
+What happens when the original repository gets updated with some sweet features you'd like in your own fork? (or if you want to avoid merge conflicts)
+
+You can update your own fork. For that, you will need to have it [cloned to your local machine](https://help.github.com/en/articles/cloning-a-repository).
+
+Using the command line, after you navigate to your cloned repository, you can first check if you have a remote reference to the upstream (original repo).
+
+```
+git remote -v
+```
+
+This will output a list of remote locations your repository is connected to.
+
+If you only see entries starting with `origin`, you do not have an upstream link and will need to create it.
+
+- Setting up an upstream remote
+
+To setup an upstream remote run the following command in your cloned repo:
+
+```
+git remote add origin https://github.com/discord-bot-tutorial/Miunie.git
+```
+
+Verify the origin was added by running the `git remote -v` command again.
```

Comment:
Weird sentence:

You can verify if the origin has been added by running `git remote -v` command once again.